spec/models/ingredient_spec.rb

```diff
@@ -2,14 +2,27 @@
 
 RSpec.describe Ingredient, type: :model do
 
-   describe "validations" do
+  describe "validations" do
       it { should validate_presence_of :name }
       it { should validate_presence_of :cost }
-   end
+  end
 
-   describe "relationships" do
+  describe "relationships" do
       it { should have_many :recipe_ingredients }
       it { should have_many(:recipes).through(:recipe_ingredients) }
-   end
+  end
 
+  describe "instance methods" do
+  	describe "#ingredients_alphabetized" do
+      it "alphabetizes all ingredients" do
+
+				ground_beef = Ingredient.create!(name: "Ground Beef", cost: 2)
+				cheese = Ingredient.create!(name: "Cheese", cost: 3)
+				tomatillo = Ingredient.create!(name: "Tomatillo", cost: 5)
+				tortilla = Ingredient.create!(name: "Tortilla", cost: 6)
+						
+				expect(Ingredient.alphabetized).to eq([cheese, ground_beef, tomatillo, tortilla])
+      end
+    end
+  end
```

Comment:
There's something wonky going on with the indentation in this file, but other than that, everything this great.